include/ginkgo/core/base/polymorphic_object.hpp

```diff
@@ -332,6 +363,11 @@ class EnableAbstractPolymorphicObject : public PolymorphicBase {
             this->copy_from_impl(std::move(other)));
     }
 
+    AbstractObject* move_from(PolymorphicObject* other)
+    {
+        return static_cast<AbstractObject*>(this->move_from_impl(other));
```

Comment:
Is the logger being called if you call move_from/copy_from on a concrete object? Maybe we should test that behavior?

Reply:
No, the logger is not called here. I think this is a general pattern in our `Enable*` mixins. I'm not fixing this here, instead I will open a new pr which addresses all instances.